### PR TITLE
fix(docs): redirects, some links.

### PR DESCRIPTION
**Pull request by nathanhammond**
Close out a few more site navigation issues.

### CODE REVIEW

File: docs/next.config.js [hunk 1]
```diff
@@ -19,7 +19,6 @@ const OLD_TURBOREPO_ROUTES = [
   "/docs/core-concepts/caching",
   "/docs/core-concepts/remote-caching",
   "/docs/core-concepts/scopes",
-  "/docs/core-concepts/why-turborepo",
```
Comment (nathanhammond) [inline docs/next.config.js:22]: This is manually addressed below since it doesn't exist in the current site.

File: docs/next.config.js [hunk 2]
```diff
@@ -92,7 +91,7 @@ const nextConfig = withNextra({
       })),
       {
         source: "/usage",
-        destination: "/reference/command-line-reference",
+        destination: "/repo/docs/reference/command-line-reference",
```
Comment (nathanhammond) [inline docs/next.config.js:94]: This is self-evidently correct; it should be scoped to the correct project.

File: docs/pages/blog/turbo-1-6-0.mdx [hunk 3]
```diff
@@ -122,7 +122,7 @@ turbo build lint
 
 Congratulations - **you just ran your first build with `turbo`**. You can try:
 
-- Running through the full [Quickstart](/repo/docs/getting-started/add-to-project).
+- Running through the full [Quickstart](/repo/docs).
```
Comment (nathanhammond) [inline docs/pages/blog/turbo-1-6-0.mdx:125]: Quickstart is the docs homepage.

File: packages/turbo/README.md [hunk 4]
```diff
@@ -25,11 +25,7 @@
 
 ## Getting Started
 
-Visit https://turbo.build/repo/docs/getting-started to get started with Turborepo.
-
-## Documentation
-
-Visit https://turbo.build/repo/docs to view the full documentation.
+Visit https://turbo.build/repo/docs to get started with Turborepo and read the documentation.
```
Comment (nathanhammond) [inline packages/turbo/README.md:28]: Making a distinction here isn't worthwhile, and linking to `/repo/docs` twice is redundant.

**Review by nathanhammond — COMMENTED** [review]
Guide for reviewers.

**Review by tknickman — APPROVED** [review]

**Pull request merged**